### ISSUE TITLE
Popup with information about recovery mode is still displayed after flashing with a valid binary

### ISSUE THREAD

**Issue by Pokisiekk** [Dasharo/dasharo-issues#320]
**Dasharo version**
v1.0.1

**Dasharo variant**
VP2420

**Affected component(s) or functionality**
internal flashing, vboot

**Brief summary**
Popup with information about recovery mode is still displayed after flashing with a valid binary.

**How reproducible**
1/1

**How to reproduce**

1. Power on the laptop.
2. Boot into Ubuntu 22.04.
3. Flash coreboot signed with wrong vboot keys by executing the following command:
   `flashrom -p internal --fmap -i RW_SECTION_A -w [coreboot_binary]`
4. Power off and then power on the laptop.
5. Boot into Ubuntu 22.04.
6. Flash dasharo firmware by executing the following command:
    a) `flashrom -p internal --fmap -i RW_SECTION_A -w [coreboot_binary]`
    b) `flashrom -p internal --ifd -i bios -w [coreboot_binary]`
7.Power off and then power on the laptop.

**Expected behavior**
The DUT boots normal, BIOS_KEYS are displayed.

**Actual behavior**
Popup with information about recovery mode is still displayed.

**Screenshots**

**Additional context**

**Solutions you've tried**
Workaround:
1. Flash dasharo firmware by executing the following command:
   `flashrom -p internal --fmap -i RW_NVRAM -w [coreboot_binary]`
2. Power off the DUT.
3. Reset CMOS.

**Comment by macpijan** [comment]
@Pokisiekk Should not we merge it with already existing issue on this subject: https://github.com/Dasharo/dasharo-issues/issues/269

What is IMHO important to mention here (and missing in this issue, but present in the linked one) is the fact that this popup appears only once - after reboot it is gone, which makes this problem less critical.

**Comment by Pokisiekk** [comment]
@macpijan I created a separate issue because, in the case of VP2420, the popup appears every time during boot after flashing `RW_SECTION` or `bios`. AFAIR, the popup disappears only after using the workaround described in the issue and then it does not appear even once, even on the first boot. In my opinion, the difference in problems is quite significant.

**Comment by miczyg1** [comment]
Fix: https://github.com/Dasharo/coreboot/pull/338

**Comment by Pokisiekk** [comment]
fix for this bug for VP2420 has been added in firmware [v1.1.0](https://docs.dasharo.com/variants/protectli_vp2420/releases/#v110-2023-04-20)
